test/parallel/test-zlib-kmaxlength-rangeerror.js

```diff
@@ -1,14 +1,18 @@
 'use strict';
-
 require('../common');
+
+// This test ensures that zlib throws a RangeError if the final buffer needs to
+// be larger than kMaxLength and concatenation fails.
+// https://github.com/nodejs/node/pull/1811
+
 const assert = require('assert');
+const buffer = require('buffer');
+const zlib = require('zlib');
 
 // Change kMaxLength for zlib to trigger the error without having to allocate
 // large Buffers.
-const buffer = require('buffer');
```

Comment:
@ryzokuken it seems that the order of these requires was part of the test. This test is now failing on all machine. Please revert this particular change.

Reply:
My bad. Fixing this in a sec.